src/org/mirah/jvm/compiler/method_state.mirah

```diff
@@ -52,9 +54,11 @@ class MethodState
       other.conflictsWith(self)
     elsif @signature.nil? || other.signature.nil?
       if @num_args == 0
-        # We know there's a conflict
-        # TODO: unless these are macros and one of them is static
-        Kind.ERROR
+        if self.isMacro && other.isMacro && (self.static ^ other.static) # unless these are macros and one of them is static
```

Comment:
How about flipping the comment to "Macros are allowed to have overlapping no-arg signatures if one is static."?